docs/reference/policy-evaluation-timeout.md

```diff
@@ -21,60 +21,58 @@ This feature is available starting from Kubewarden v1.5.0.
 It's purpose is to limit the amount of time a request evaluation can take.
 
-This feature is enabled by default from Kubewarden v1.5.0.
+Kubewarden enables this feature from version v1.5.0.
 
 ## Purpose
 
-Kubewarden policies can be written using both traditional programming languages
+You can write Kubewarden policies using both traditional programming languages
 (like [Go](../tutorials/writing-policies/go/01-intro-go.md),
 [Rust](../tutorials/writing-policies/rust/01-intro-rust.md) and
-[others](../tutorials/writing-policies/index.md)
-) or using the special query language [Rego](../tutorials/writing-policies/rego/01-intro-rego.md).
-While both approaches have their pros and cons, the goal of Kubewarden is to allow the policy
-authors to pick the best tool to do their job.
-
-When using a traditional programming language (or, to be
-more correct, a [Turing-complete](https://en.wikipedia.org/wiki/Turing_completeness)
-language), it is possible to introduce mistakes like
-[infinite loops](https://en.wikipedia.org/wiki/Infinite_loop),
-[deadlocks](https://en.wikipedia.org/wiki/Deadlock) or code that runs slowly
-because it lacks optimizations or simply because it performs computationally
-intense operations.
-
-The policy evaluation timeout protection feature terminates the evaluation of
-a request after a certain time. This ensures Policy Server always has compute
-resources available to process incoming requests.
+[others](../tutorials/writing-policies/index.md)) or using the special query
+language [Rego](../tutorials/writing-policies/rego/01-intro-rego.md). Both
+approaches have merits so a goal of Kubewarden is to let the policy authors
+choose the best tool for their needs.
+
+When using a traditional, [Turing-complete](https://en.wikipedia.org/wiki/Turing_completeness) programming language, it's possible to have issues like:
+
+* [infinite loops](https://en.wikipedia.org/wiki/Infinite_loop)
+* [deadlocks](https://en.wikipedia.org/wiki/Deadlock).
+* slow running code lacking optimizations
+* computationally intense operations.
+
+The policy evaluation timeout protection feature terminates the evaluation of a
+request after a defined period of time. This ensures Policy Server always has
+compute resources available to process incoming requests.
 
 ## Limitations
 
-Currently, policy evaluation timeout protection is capable of interrupting
-the majority of long running evaluations.
-There are however certain edge cases that are not yet handled. This includes
-invoking a `sleep` instruction from within a policy and deadlocks.
+Currently, policy evaluation timeout protection is capable of interrupting most
+long running evaluations. There are certain edge cases not yet handled.
+This includes invoking a `sleep` instruction from within a policy, and
+deadlocks.
 
-These scenarios are going to be handled by a future release of Policy Server.
+A future release of Policy Server will address these scenarios.
```

Comment:
I think we can remove this sentence. We do not have plans for that now. Considering that no user requested that, we can wait a little bit more until this turn into a priority.